### ISSUE TITLE
Accessibility Issues in TeamTalk Mac Client

### ISSUE THREAD

**Issue by Muamalaljanahi** [BearWare/TeamTalk5#2329]
### Description

Hello,
The TeamTalk client for Mac is experiencing several accessibility issues. These include:
1. Often, the names of users/channels are not read when hovering over them with the keyboard cursor, as well as the server list.
2. Channel messages are completely unreadable with VoiceOver, and sometimes even private messages if there are many messages. This might be resolved in Pull Request #2310 
3. Additional minor issues here and there.

These issues make TeamTalk somewhat unusable with VoiceOver.
Any assistance on how to handle or avoid these issues would be greatly appreciated.
Thank you very much.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [X] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

VoiceOver should Always read the elements as it does with other applications.

### Actual behavior

VoiceOver does not read the elements, especially when expanding or collapsing the tree view or when someone joins the server. This makes using TeamTalk a challenging task for VoiceOver users, which is surprising since TeamTalk is highly compatible on other systems such as Windows, iOS, and Android.

### Steps to reproduce problem

Try to navigate the channel list and the channel message field with VoiceOver enabled

**Comment by CoBC** [comment]
All those issues are present for a while and are in majority related to Qt accessibility issues. It's really $hard to voerrride Qt classto reimplement accessibility features. I'm trying to play with QAccessible, QAccessibleInterface and other accessibility class currently but for now the result is not very concluent.
This answer is especially right for first point, for second point, you're right, this will probably be fix if the PR #2310 is accept.
